jwt/validate_test.go

```diff
@@ -256,3 +257,63 @@ func TestGHIssue10(t *testing.T) {
 		}
 	})
 }
+
+type validateFn func(interface{}) error
```

Comment:
This should be an exported type. Then the consumers of the API can just convert types on the fly like you do in the test
```
type ClaimValidatorFunc func(interface{}) error
```